control_panel_api/services.py

```diff
@@ -153,6 +153,13 @@ def delete_bucket_policies(bucket_name):
     aws.delete_policy(policy_arn_readonly)
 
 
+def app_granted_access_to_bucket(app_slug, bucket_name, readwrite):
```

Comment:
Following the convention from other service calls that are called by the view, this should be named `service.apps3bucket_create`

Reply:
I'm not really sure about this naming convention as the service layer names should be driven by the domain problem in my mind - Not by whatever we called the models (especially in the case of this one which it's not exactly great).

This `services` module is starting to look more and more like `callbacks` rather than a service layer with a set of high level operations you can call.

Rename this function is obviously trivial but we should probably have a quick chat about it.

Reply:
Even if I think long term we should try to keep the service layer more high-level, I'm changing it to `app_s3bucket` (as it's the relation between apps and S3 buckets) 👍